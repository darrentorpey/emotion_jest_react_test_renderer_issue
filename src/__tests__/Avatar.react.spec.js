import React from 'react';
import Avatar from '../Avatar.react';
import renderer from 'react-test-renderer';

test('An Avatar can be rendered', () => {
  const component = renderer.create(
    <Avatar>Hi <img /></Avatar>
  );

  expect(component.toJSON()).toMatchSnapshot();
});
