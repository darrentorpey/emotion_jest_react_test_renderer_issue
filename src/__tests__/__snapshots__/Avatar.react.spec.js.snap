// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`An Avatar can be rendered 1`] = `
<div
  css="
  & img {
    width: 96px;
    height: 96px;
    border-radius: 50%;
  }
"
/>
`;
